desktop/settings/windows.md

```diff
@@ -200,12 +200,33 @@ For more details on configuring Docker Desktop to use WSL 2, see
 
 ## Docker Engine
 
-The **Docker Engine** tab allows you to configure the Docker daemon to determine how your containers run.
-
-Type a JSON configuration file in the box to configure the daemon settings. For a full list of options, see the Docker Engine
-[dockerd commandline reference](/engine/reference/commandline/dockerd/){:target="_blank" rel="noopener" class="_"}.
-
-Click **Apply & Restart** to save your settings and restart Docker Desktop.
+The **Docker Engine** tab allows you to configure the Docker daemon used to run containers with Docker Desktop.
+
+You configure the daemon using a JSON configuration file. Here's what the file might look like:
+
+```json
+{
+  "builder": {
+    "gc": {
+      "defaultKeepStorage": "20GB",
+      "enabled": true
+    }
+  },
+  "experimental": false,
+  "features": {
+    "buildkit": true
+  }
+}
+```
+
+You can find this file at `$HOME/.docker/daemon.json`. To change the configuration, either
```

Comment:
Probably okay for a follow-up, but we need to check what config location is used for the "native windows containers" daemon on Windows; does it share the config, or does it use the default (`C:\ProgramData\docker\config\daemon.json`) location?